docs/concepts/customizing-execution.rst

```diff
@@ -35,57 +45,47 @@ something.
             """This function builds the result given the computed values."""
             pass
 
-So we have a few implementations see :doc:`../../reference/result-builders/index` for the list.
-
-To use it, it needs to be paired with a GraphAdapter - onto the next section!
+So we have a few implementations see :doc:`../../reference/result-builders/index` for the list. These can be run independently,
+or paired with a graph adapter, see next section for details!
 
 Graph Adapters
 ##############
 
 Graph Adapters `adapt` the Hamilton DAG, and change how it is executed. They all implement a single interface called
 ``base.HamiltonGraphAdapter``. They are called internally by Hamilton at the right points in time to make execution
-work. The link with the Result Builders, is that GraphAdapters need to implement a ``build_result()`` function
+work. The link with the Result Builders, is that GraphAdapters (currently) need to implement a ``build_result()`` function
 themselves.
 
 .. code-block:: python
 
-    class HamiltonGraphAdapter(ResultMixin):
+    class HamiltonGraphAdapter(customization.ResultBuilder):
```

Comment:
I think `lifecycle` is a better name than `customization`?

Reply:
Lifecycle is nebulous -- customization is what we're doing?

Reply:
Changed to `lifecycle` (we discussed offline) -- more consistent and descriptive enough